test/image/mocks/log-axis_no-minor_suffix-prefix.json

```diff
@@ -0,0 +1,117 @@
+{
+  "data": [
+    {
+      "y": [
```

Comment:
Could you set the trace `name` to their corresponding `yaxis?.dtick` value to improve the mock?

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/4064/commits/cb521f5d29e539527075a54c1900c5b232a19791.